mod/developers/views/default/admin/develop_tools/inspect/services.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+$data = elgg_extract("data", $vars);
+
+if (empty($data)) {
+	return;
+}
+
+echo "<table class='elgg-table-alt'>";
+echo "<tr>";
+echo "<th>" . elgg_echo('developers:inspect:service:name') . "</th>";
```

Comment:
you should add thead for better styling of th

Reply:
I know, noticed that later. But there are a ton of forms that need rework. Leaving as is for now.